dev/integration_tests/flutter_gallery/lib/demo/colors_demo.dart

```diff
@@ -96,6 +96,7 @@ class PaletteTabView extends StatelessWidget {
     final TextStyle blackTextStyle = textTheme.bodyText2!.copyWith(color: Colors.black);
     return Scrollbar(
       child: ListView(
+        primary: true,
```

Comment:
These integration tests are run on mobile and desktop platforms, so I have added `primary: true` so they all inherit the PrimaryScrollController as they did so automatically prior to this change.

Reply:
Only on desktop, which is the point of this change.

The automatic attachment of the PrimaryScrollController will no longer happen on desktop. Thee integration test expect that behavior across all of the platforms. The tests could be refactored for each platform, but that seemed outside of the scope for this change.

Reply:
Without this, the test will fail when run configured to desktop, because it is no longer automatic.